consensus.go

```diff
@@ -0,0 +1,238 @@
+package dcrlibwallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+
+	"decred.org/dcrwallet/v2/errors"
+	w "decred.org/dcrwallet/v2/wallet"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrd/chaincfg/v3"
+	"github.com/decred/dcrd/wire"
+)
+
+const (
+	DcrdataMainnetHost = "https://dcrdata.decred.org/api/agendas"
+	DcrdataTestnetHost = "https://testnet.decred.org/api/agendas"
+
+	AgendaStatusUpcoming   = "upcoming"
+	AgendaStatusInProgress = "in progress"
+	AgendaStatusFinished   = "finished"
```

Comment:
Not sure these are used. Are they?

Reply:
https://github.com/planetdecred/godcr/pull/729/files#diff-dec8e3ea6e8e79fc4373da512c8399413ebc858af71ca72a673e3739a2113e8aR45 yeah, they are used